modules/claims_api/app/swagger/claims_api/v0/form_2122_controller_swagger.rb

```diff
@@ -352,7 +360,103 @@ class Form2122ControllerSwagger
             key :description, '2122 response'
             content 'application/json' do
               schema do
-                key :'$ref', :Form2122Output
+                key :type, :object
+                key :required, [:data]
+                property :data do
+                  key :'$ref', :Form2122Output
+                end
+              end
+            end
+          end
+
+          response :default do
+            key :description, 'unexpected error'
+            content 'application/json' do
+              schema do
+                key :type, :object
+                key :required, [:errors]
+                property :errors do
+                  key :type, :array
+                  items do
+                    key :'$ref', :ErrorModel
+                  end
+                end
+              end
+            end
+          end
+        end
+      end
+
+      swagger_path '/forms/2122/active' do
```

Comment:
this endpoint was missing from the v0 docs. from what I could tell, the logic is similar enough to v1 that the responses should be the same